lib/models/file-info.js

```diff
@@ -95,7 +97,33 @@ var FileInfo = CoreObject.extend({
     this._fileContents = fse.readFileSync(fullPath, { encoding: 'utf8' });
   },
 
+  updateImports: function() {
+    if (this.ext !== '.js' || !this._fileContents) { return; } // only process JavaScript files
+
+    var appName = this.options.projectName;
+
+    try {
+      var newContents = importDeclarationsTransform(
+        { source: this._fileContents,
+          fileInfo: this,
+          appName: appName,
+          fileInfos: this._fileInfoCollection._fileInfos
+        },
+        { jscodeshift: jscodeshift });
+
+      var fullPath = path.join(this.projectRoot, this.sourceRelativePath);
+
+      fse.writeFileSync(fullPath, newContents, { encoding: 'utf-8' });
+      this._fileContents = fse.readFileSync(fullPath, { encoding: 'utf8' });
```

Comment:
So, it appears that part is copied from updateDefaultExportToNamed (that you authored!  https://github.com/rwjblue/ember-module-migrator/commit/b80eb998ffcdefb9706a72cf2660b1e80f98f904). I'll try refactoring both and seeing if the tests still pass, since it indeed looks unnecessary